src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumDistributePointsType.java

```diff
@@ -0,0 +1,23 @@
+package teammates.common.datatransfer.questions;
+
+/**
+ * Enum that defines different distribute points options for constant sum questions.
+ */
+public enum FeedbackConstantSumDistributePointsType {
+    DISTRIBUTE_ALL_UNEVENLY("All options"),
+    DISTRIBUTE_SOME_UNEVENLY("At least some options"),
+    NONE("None");
```

Comment:
I like the use of enum:D but i think maybe u can put these strings into Const as they will be reused in quite some diff places?